src/modules/vtol_att_control/standard_params.c

```diff
@@ -43,35 +43,34 @@
 /**
  * Use fixed-wing actuation in hover to accelerate forward
  *
- * This feature can be used to avoid the plane having to pitch nose down in order to move forward.
- * Prevents large, negative lift from pitching nose down into wind.
- * Fixed-wing forward actuators refers to puller/pusher (standard VTOL), or forward-tilt (tiltrotor VTOL).
- * Only active if demanded down pitch is below VT_PITCH_MIN.
+ * Prevents downforce from pitching the body down when facing wind.
+ * Uses puller/pusher (standard VTOL), or forward-tilt (tiltrotor VTOL) to accelerate forward instead.
+ * Only active if demanded pitch  is below VT_PITCH_MIN.
  * Use VT_FWD_THRUST_SC to tune it.
  * Descend mode is treated as Landing too.
  *
- * Only active (if enabled) in Altitude, Position and Auto modes, not in Stabilized.
+ * Only active (if enabled) in height-rate controlled modes.
  *
  * @value 0 Disabled
```

Comment:
"On" and "Off" would be even shorter. Also all options except for the first start with "Enabled". But if it fits the UI now, then there's no need to change.